proto/agglayer/prover/v1/proof_generation.proto

```diff
@@ -43,3 +43,33 @@ message GenerateProofError {
   // The kind of error that occurred.
   ErrorKind error_type = 2;
 }
+
+// Service for fetching auth proof.
+service AuthProofService {
+  // Fetches a auth-proof for a given start_block.
+  rpc FetchAuthProof(FetchAuthProofRequest) returns (FetchAuthProofResponse);
+}
+
+message FetchAuthProofRequest {
+  // The start block for which the auth proof is requested.
+  uint64 start_block = 1;
+  // The max end block for which the auth proof is requested.
+  uint64 max_end_block = 2;
+}
+
+message FetchAuthProofResponse {
+  // The auth proof.
+  // TODO - Define the type of auth proof.
+  bytes auth_proof = 1;
```

Comment:
@Freyskeyd whats the expected type for auth_proof?

Reply:
It's currently undefined